homeassistant/components/synology_dsm/const.py

```diff
@@ -95,6 +97,13 @@ class SynologyDSMBinarySensorEntityDescription(
     """Describes Synology DSM binary sensor entity."""
 
 
+@dataclass
+class SynologyDSMUpdateEntityEntityDescription(
```

Comment:
This should be defined in `update.py` ?
Same with the others really, they are tied to their specific platforms.

Reply:
> Same with the others really, they are tied to their specific platforms.

yeah I know ... the whole integration need some love to be reworked 🙈